fix:[CDS-88785]: adding log limitation ref in approval logs (#5262)
harness/developer-hub: docs/continuous-delivery/shared/jira-approval-steps.md

![](./static/adding-jira-approval-stages-14.png)

> - Please refer to [Document on deloyment logs and limitations](https://developer.harness.io/docs/continuous-delivery/manage-deployments/deployment-logs-and-limitations) for limitations of approval logs. 

Reviewer: ```suggestion
- For more information about approval log limitations, go to [Deployment logs and limitations](/docs/continuous-delivery/manage-deployments/deployment-logs-and-limitations). 
```
